amundsen_application/static/js/components/SearchPage/ResourceSelector/index.tsx

```diff
@@ -32,7 +39,7 @@ export class ResourceSelector extends React.Component<ResourceSelectorProps > {
   }
 
   onChange = (event) => {
-    this.props.onChange(event.target.value);
+    this.props.setResource(event.target.value);
```

Comment:
nit: We should still have a test for this that verifies this does what it's supposed to do. Would require making a mockEvent and using @ts-ignore to avoid mocking the whole event -- a couple examples exist in the `SearchBar` tests on this branch. (a backlog ticket is already filed for a utility to create type safe mock events for testing)